src/StoreApi/Schemas/CartSchema.php

```diff
@@ -192,13 +198,13 @@ public function get_properties() {
 						],
 						'total_shipping'     => [
 							'description' => __( 'Total price of shipping.', 'woo-gutenberg-products-block' ),
-							'type'        => 'string',
+							'type'        => [ 'string', 'null' ],
```

Comment:
Should we indicate what `null` means in the response here?

Reply:
Added